### PR TITLE
[ci skip] Add `render_collection.action_view` entry to AS instrumenta…

### PR DESCRIPTION
**Pull request by yui-knk**

### CODE REVIEW

File: guides/source/active_support_instrumentation.md [hunk 1]
```diff
@@ -226,6 +226,20 @@ Action View
 }
 ```
 
+### render_collection.action_view
+
+| Key           | Value                 |
+| ------------- | --------------------- |
+| `:identifier` | Full path to template |
+| `:count`      | Size of collection    |
```
Comment (kaspth) [inline guides/source/active_support_instrumentation.md]: There's also an optional `:cache_hits` key for the collections rendered with `cached: true`: https://github.com/rails/rails/blob/1de3041e3be7be8477ee28c7ece616a5975321bd/actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb#L17

File: guides/source/active_support_instrumentation.md [hunk 2]
```diff
@@ -226,6 +226,20 @@ Action View
...
+
+```ruby
+{
+  identifier: "/Users/adam/projects/notifications/app/views/posts/index.html.erb",
```
Comment (vipulnsward) [inline guides/source/active_support_instrumentation.md]: This is still a partial, so something like "/Users/adam/projects/notifications/app/views/posts/_comment.html.erb" would be a better name here.

Comment (kaspth) [inline guides/source/active_support_instrumentation.md]: `posts/_post.html.erb` would be even better 😄

**Comment by yui-knk** [comment]
@kaspth Thanks for your review ❤️ I updated.

**Comment by kaspth** [comment]
Great, thanks!

@vipulnsward Thanks for the extra pair of eyes as well 👀

**Comment by kaspth** [comment]
The event also exists on 5-0-stable, so I backported to there as well: be798fa (but don't think we should reach for 4.2 if the event is there).

**Pull request merged**